### ISSUE TITLE
Success icon has no translation in system message

### ISSUE THREAD

**Issue by Sulpher** [joomla/joomla-cms#47861]
### Steps to reproduce the issue

When Bootstrap disabled, the icon has no translation in the system message (you are successfully logged in). Have a look:

<img width="2070" height="338" alt="Image" src="https://github.com/user-attachments/assets/5df4834b-111f-4c06-aad0-f893d9fe9825" />

According to this discussion with Russian localization team: https://github.com/JPathRu/localisation/issues/135#issuecomment-4585916960
I am reporting this issue here.

The problem is in Cassiopeia layout:

<img width="551" height="358" alt="Image" src="https://github.com/user-attachments/assets/5bc08d4a-1119-42d4-a4af-adffe1e2de2a" />

### Version

5.4

### Expected result

The success icon should has translation

### Actual result

Unable to localize icon name which is important for accessibility.
